stdlib/2/typing.pyi

```diff
@@ -69,7 +72,8 @@ _KT_co = TypeVar('_KT_co', covariant=True)  # Key type covariant containers.
 _VT_co = TypeVar('_VT_co', covariant=True)  # Value type covariant containers.
 _T_contra = TypeVar('_T_contra', contravariant=True)  # Ditto contravariant.
 _TC = TypeVar('_TC', bound=Type[object])
-_C = TypeVar("_C", bound=Callable[..., Any])
+# _C is now declared earlier for no_type_check_decorator
```

Comment:
No need to keep this comment around. Although maybe we should just keep all typevars to the top to keep them together.